cowprotocol/cowswap
# 332/warning when pinata envs not set locally

# Summary

Part of #332 

Adding warning locally when PINATA keys are not set

<img width="994" alt="Screen Shot 2022-06-08 at 14 41 03" src="https://user-images.githubusercontent.com/43217/172633147-fb913e67-fad8-4fa6-85af-9681ce309e13.png">

## Edit 1

Added another alternative approach as suggested by David
<img width="373" alt="Screen Shot 2022-06-08 at 16 21 42" src="https://user-images.githubusercontent.com/43217/172655762-dd7e3a31-2884-450b-92fe-e71f4d77d989.png">

Of course will leave only one, but right now both are active for comparison

  # To Test

1. Spin it up locally without setting pinata env keys
* You should see the warning
2. Set up the required env vars
* Warning will be gone
